I would also like to let you know that the update script does not work for me in case of `signal-desktop-beta`
```
export UPDATE_NIX_ATTR_PATH='signal-desktop-beta' && ./pkgs/applications/networking/instant-messengers/signal-desktop/update.sh
``` 

```
+++ dirname ./pkgs/applications/networking/instant-messengers/signal-desktop/update.sh
++ cd ./pkgs/applications/networking/instant-messengers/signal-desktop
++ pwd
+ SCRIPT_DIR=/home/prometheus/Nixpkgs/nixpkgs/pkgs/applications/networking/instant-messengers/signal-desktop
+ case "$UPDATE_NIX_ATTR_PATH" in
++ curl_github https://api.github.com/repos/signalapp/Signal-Desktop/releases
++ curl https://api.github.com/repos/signalapp/Signal-Desktop/releases
++ jq -r '.[0].tag_name'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 81398    0 81398    0     0  77762      0 --:--:--  0:00:01 --:--:-- 77818
+ latestTagBeta=v6.42.0-beta.1
++ expr v6.42.0-beta.1 : 'v\(.*\)'
+ latestVersionBeta=6.42.0-beta.1
+ echo 'Updating signal-desktop-beta for x86_64-linux'
Updating signal-desktop-beta for x86_64-linux
+ nix-update --version 6.42.0-beta.1 --system x86_64-linux --override-filename /home/prometheus/Nixpkgs/nixpkgs/pkgs/applications/networking/instant-messengers/signal-desktop/signal-desktop-beta.nix signal-desktop-beta
$ nix eval --json --impure --expr 
let
  
  pkgs = import "/home/prometheus/Nixpkgs/nixpkgs";
  args =  builtins.functionArgs pkgs;
  inputs = (if args ? system then { system = "x86_64-linux"; } else {}) //
           (if args ? overlays then { overlays = [ ]; } else {});
  pkg = (pkgs inputs)."signal-desktop-beta";
  sanitizePosition = x: x;

  raw_version_position = sanitizePosition (builtins.unsafeGetAttrPos "version" pkg);

  position = if pkg ? isRubyGem then
    raw_version_position
  else if pkg ? isPhpExtension then
    raw_version_position
   else
    sanitizePosition (builtins.unsafeGetAttrPos "src" pkg);
in {
  name = pkg.name;
  old_version = pkg.version or (builtins.parseDrvName pkg.name).version;
  inherit raw_version_position;
  filename = position.file;
  line = position.line;
  urls = pkg.src.urls or null;
  url = pkg.src.url or null;
  rev = pkg.src.rev or null;
  hash = pkg.src.outputHash or null;
  go_modules = pkg.goModules.outputHash or null;
  go_modules_old = pkg.go-modules.outputHash or null;
  cargo_deps = pkg.cargoDeps.outputHash or null;
  raw_cargo_lock =
    if pkg ? cargoDeps.lockFile then
      let
        inherit (pkg.cargoDeps) lockFile;
        res = builtins.tryEval (sanitizePosition {
          file = toString lockFile;
        });
      in
      if res.success then res.value.file else false
    else
      null;
  npm_deps = pkg.npmDeps.outputHash or null;
  yarn_deps = pkg.offlineCache.outputHash or null;
  tests = builtins.attrNames (pkg.passthru.tests or {});
  has_update_script = pkg.passthru.updateScript or null != null;
  src_homepage = pkg.src.meta.homepage or null;
  changelog = pkg.meta.changelog or null;
} --system x86_64-linux --extra-experimental-features flakes nix-command
error:
       … while evaluating attribute 'filename'

         at «string»:23:3:

           22|   inherit raw_version_position;
           23|   filename = position.file;
             |   ^
           24|   line = position.line;

       … while calling the 'unsafeGetAttrPos' builtin

         at «string»:18:23:

           17|    else
           18|     sanitizePosition (builtins.unsafeGetAttrPos "src" pkg);
             |                       ^
           19| in {

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: value is a path while a set was expected
Traceback (most recent call last):
  File "/nix/store/rss9z6n3rki1xhzlzgq47c7z5qwq3ikx-nix-update-1.0.0/bin/.nix-update-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/rss9z6n3rki1xhzlzgq47c7z5qwq3ikx-nix-update-1.0.0/lib/python3.11/site-packages/nix_update/__init__.py", line 287, in main
    package = update(options)
              ^^^^^^^^^^^^^^^
  File "/nix/store/rss9z6n3rki1xhzlzgq47c7z5qwq3ikx-nix-update-1.0.0/lib/python3.11/site-packages/nix_update/update.py", line 322, in update
    package = eval_attr(opts)
              ^^^^^^^^^^^^^^^
  File "/nix/store/rss9z6n3rki1xhzlzgq47c7z5qwq3ikx-nix-update-1.0.0/lib/python3.11/site-packages/nix_update/eval.py", line 181, in eval_attr
    res = run(cmd)
          ^^^^^^^^
  File "/nix/store/rss9z6n3rki1xhzlzgq47c7z5qwq3ikx-nix-update-1.0.0/lib/python3.11/site-packages/nix_update/utils.py", line 37, in run
    return subprocess.run(
           ^^^^^^^^^^^^^^^
  File "/nix/store/qp5zys77biz7imbk6yy85q5pdv7qk84j-python3-3.11.6/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['nix', 'eval', '--json', '--impure', '--expr', '\nlet\n  \n  pkgs = import "/home/prometheus/Nixpkgs/nixpkgs";\n  args =  builtins.functionArgs pkgs;\n  inputs = (if args ? system then { system = "x86_64-linux"; } else {}) //\n           (if args ? overlays then { overlays = [ ]; } else {});\n  pkg = (pkgs inputs)."signal-desktop-beta";\n  sanitizePosition = x: x;\n\n  raw_version_position = sanitizePosition (builtins.unsafeGetAttrPos "version" pkg);\n\n  position = if pkg ? isRubyGem then\n    raw_version_position\n  else if pkg ? isPhpExtension then\n    raw_version_position\n   else\n    sanitizePosition (builtins.unsafeGetAttrPos "src" pkg);\nin {\n  name = pkg.name;\n  old_version = pkg.version or (builtins.parseDrvName pkg.name).version;\n  inherit raw_version_position;\n  filename = position.file;\n  line = position.line;\n  urls = pkg.src.urls or null;\n  url = pkg.src.url or null;\n  rev = pkg.src.rev or null;\n  hash = pkg.src.outputHash or null;\n  go_modules = pkg.goModules.outputHash or null;\n  go_modules_old = pkg.go-modules.outputHash or null;\n  cargo_deps = pkg.cargoDeps.outputHash or null;\n  raw_cargo_lock =\n    if pkg ? cargoDeps.lockFile then\n      let\n        inherit (pkg.cargoDeps) lockFile;\n        res = builtins.tryEval (sanitizePosition {\n          file = toString lockFile;\n        });\n      in\n      if res.success then res.value.file else false\n    else\n      null;\n  npm_deps = pkg.npmDeps.outputHash or null;\n  yarn_deps = pkg.offlineCache.outputHash or null;\n  tests = builtins.attrNames (pkg.passthru.tests or {});\n  has_update_script = pkg.passthru.updateScript or null != null;\n  src_homepage = pkg.src.meta.homepage or null;\n  changelog = pkg.meta.changelog or null;\n}', '--system', 'x86_64-linux', '--extra-experimental-features', 'flakes nix-command']' returned non-zero exit status 1.
```

--- a/pkgs/applications/networking/instant-messengers/signal-desktop/default.nix
+++ b/pkgs/applications/networking/instant-messengers/signal-desktop/default.nix
@@ -2,5 +2,5 @@
   signal-desktop = if hostPlatform.system == "aarch64-linux"
     then callPackage ./signal-desktop-aarch64.nix { }
     else callPackage ./signal-desktop.nix { };
-  signal-desktop-beta = ./signal-desktop-beta.nix;
+  signal-desktop-beta = callPackage ./signal-desktop-beta.nix{ };
 }

--- a/pkgs/applications/networking/instant-messengers/signal-desktop/update.sh
+++ b/pkgs/applications/networking/instant-messengers/signal-desktop/update.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p bash nix-update curl coreutils
+#!nix-shell -i bash -p bash nix-update curl coreutils jq
 
 set -ex
 
@@ -18,7 +18,7 @@ signal-desktop)
   echo "Updating signal-desktop for x86_64-linux"
   nix-update --version "$latestVersion" \
     --system x86_64-linux \
-    --override-filename "$SCRIPT_DIR/signal-desktop.nix"
+    --override-filename "$SCRIPT_DIR/signal-desktop.nix" \
   signal-desktop
 
   echo "Updating signal-desktop for aarch64-linux"

